xfer/resolver.go

```diff
@@ -1,4 +1,4 @@
-package main
+package xfer
```

Comment:
It just didn't seem to 'fit' in the probe package; it was much more xfer-y, don't you think?

In other words: no good reason.